code/modules/telesci/bcrystal.dm

```diff
@@ -0,0 +1,71 @@
+// Bluespace crystals, used in telescience and when crushed it will blink you to a random turf.
+
+/obj/item/bluespace_crystal
+	name = "bluespace crystal"
+	desc = "A glowing bluespace crystal, not much is known about how they work. It looks very delicate."
+	icon = 'icons/obj/telescience.dmi'
+	icon_state = "bluespace_crystal"
+	w_class = 1
+	origin_tech = list(TECH_BLUESPACE = 4, TECH_MATERIAL = 3)
+	var/blink_range = 8 // The teleport range when crushed/thrown at someone.
+
+
+/obj/item/bluespace_crystal/New()
+	..()
+	pixel_x = rand(-5, 5)
+	pixel_y = rand(-5, 5)
+//	create_reagents(10)
+//	reagents.add_reagent("bluespace_dust", blink_range)
+
+/obj/item/bluespace_crystal/attack_self(mob/user)
+	user.visible_message("<span class='warning'>[user] crushes [src]!</span>", "<span class='danger'>You crush [src]!</span>")
```

Comment:
use SPAN_WARNING()

Reply:
fix this also for all messages containing `<span class='warning'>` `<span class='notice'>` etc...